wa/starter/pom.xml

```diff
@@ -43,13 +43,16 @@ under the License.
       <artifactId>syncope-wa-bootstrap</artifactId>
       <version>${project.version}</version>
     </dependency>
-
     <dependency>
       <groupId>org.apache.syncope.common.keymaster</groupId>
       <artifactId>syncope-common-keymaster-client-api</artifactId>
       <version>${project.version}</version>
     </dependency>
-
+    <dependency>
+      <groupId>org.apache.syncope.core</groupId>
+      <artifactId>syncope-core-provisioning-api</artifactId>
```

Comment:
`syncope-core-provisioning-api` are meant only for usage by `Core`: the relevant class(es) should be extracted and placed into `common/idrepo/lib`.